client/src/types/signup.ts

```diff
@@ -4,8 +4,8 @@ export interface Signup {
   lastName: string
   email: string
   password: string
-  confirmPassword: string
-  avatar: string
+  confirmPassword?: string // gonna remove before sending to server
+  avatar?: string // ^^^
```

Comment:
The comments can be removed if this has been handled and a user should always have an Avata